openmdao/components/indep_var_comp.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, name, val=None, **kwargs):
                              "`str` or an iterable of tuples of the form (name, value) or "
                              "(name, value, keyword_dict).")
 
-    def apply_linear(self, mode, ls_inputs=None, vois=(None, ), gs_outputs=None):
+    def sys_apply_linear(self, mode, ls_inputs=None, vois=(None, ), gs_outputs=None):
```

Comment:
Fair question. Regardless I think that the `sys` should stay as it differentiates the function clearly from the user-hook. 

It is part of the public api of system, group, and component. Since other classes will be calling that method on it, I think it should not have the underscore.